src/components/DappBrowser/Dimensions.ts

```diff
@@ -2,11 +2,10 @@ import { IS_ANDROID, IS_IOS } from '@/env';
 import { TAB_BAR_HEIGHT } from '@/navigation/SwipeNavigator';
 import { deviceUtils, safeAreaInsetValues } from '@/utils';
 import { StatusBar } from 'react-native';
-import { NAVIGATION_BAR_HEIGHT } from '@/utils/deviceUtils';
 
 export const BOTTOM_BAR_HEIGHT = 88;
 export const TOP_INSET = IS_IOS ? safeAreaInsetValues.top : StatusBar.currentHeight ?? 40;
-export const BOTTOM_INSET = IS_ANDROID ? NAVIGATION_BAR_HEIGHT - 8 : BOTTOM_BAR_HEIGHT;
+export const BOTTOM_INSET = IS_ANDROID ? safeAreaInsetValues.bottom + 32 : BOTTOM_BAR_HEIGHT;
```

Comment:
Some fixes for Android webview height now that we added some padding to the bottom of the entire app due to safe view